axelor-prestashop/src/main/resources/i18n/messages_fr.csv

```diff
@@ -1,43 +1,102 @@
 "key","message","comment","context"
+"<p>WARNING: Order has been modified on PrestaShop but could not be updated locally.</p>","<p>ATTENTION : la commande a été modifiée sur PrestaShop mais les modifications n'ont pas pu être répercutées.</p>",,
 "Action",,,
-"App prestashop","App Prestashop",,
-"Awaiting Cash On Delivery validation",,,
-"Awaiting bank wire payment",,,
-"Awaiting check payment",,,
-"Batchs",,,
-"Canceled",,,
+"An HTTP error occured while checking access rights: %s","Une erreur est survenue lors de la vérification des permissions : %s",,
+"An error occured while checking Prestashop access rights: %s, see server logs for details","Une erreur est survenue lors de la vérification des permissions : %s, consultez les fichiers journaux pour plus de détails",,
+"Anomalies",,,
+"App prestashop",,,
+"Axelor texts languages (for descriptions of products, name of countries, aso.)","Langue des textes sous Axelor (description des produits, noms des pays, etc.)",,
+"Batch","Traitement par lot",,
+"Batch %s unknown","Traitement par lot %s inconnu",,
+"Batches","Traitements par lot",,
 "Code",,,
-"Company",,,
-"Created by",,,
-"Created on",,,
+"Company","Raison sociale",,
```

Comment:
We have to be carreful with this kind of translation. Translations will override all translations of the dependant modules. So here, all fields / Object / Menus... "Company" will be translated into "Raison social". If a translation should be different in this module, you have to define a new key.

Reply:
Done